test/dynamo/test_higher_order_ops.py

```diff
@@ -1038,12 +1038,17 @@ def false_fn(x):
         mod_for_compile = torch.compile(Foo(), backend=cnt, dynamic=True)
         mod_for_eager = Foo()
 
+        ref = torch.tensor(72.0)
```

Comment:
Does this get used anywhere?

Reply:
yeah, it's used to compare with mod_for_compile results in L1052